target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -624,12 +621,6 @@ ath79_setup_macs()
 	dlink,dap-3662-a1)
 		label_mac=$(mtd_get_mac_ascii bdcfg "wlanmac")
 		;;
-	dlink,dir-825-b1|\
-	trendnet,tew-673gru)
```

Comment:
`trendnet,tew-673gru` still needs this.
In the other conditions only remove the `dlink,dir-825-b1`.